api/lib/searchOperators.js

```diff
@@ -100,10 +155,12 @@ module.exports = {
         parameters: [modifiedSearchQuery],
       };
     }
-    return {
-      condition: 'v.FirstLetterStr BETWEEN ? AND ?',
+    const result = {
+      columns: ' LEFT JOIN tokenized_firstletters t ON t.verseid = v.ID',
```

Comment:
unnecessary space after ' or that's intentional?

Reply:
That's intentional, adding space when it combined.